### PR TITLE
Immediately process direct debits

### PR DESCRIPTION
**Pull request by lera**
[Addresses](https://kontist.atlassian.net/browse/COM-1476)

All created direct debits were not processed and stuck in the queue

### CODE REVIEW

File: src/routes/transactions.ts [hunk 1]
```diff
@@ -45,7 +47,7 @@ export const createSepaDirectDebit = async (req, res) => {
   const technicalPerson = await getTechnicalUserPerson();
```
Comment (p-janik) [inline src/routes/transactions.ts:48]: findPersonByAccountIBAN is quite slow I think, it does this super slow `getAllPersons` under the hood.

I think we could improve performance a lot if we would create different key value maps (IBAN to person id etc)

File: src/routes/transactions.ts [hunk 2]
```diff
@@ -84,23 +92,28 @@ export const createSepaDirectDebit = async (req, res) => {
...
+      return_transaction_id: booking.transaction_id,
     };
-    person.queuedBookings.push(directDebitReturn);
+    person.transactions.push(directDebitReturn);
+    const sepaDirectDebitReturn = createSepaDirectDebitReturn(
+      person,
+      directDebitReturn
+    );
+    await saveSepaDirectDebitReturn(sepaDirectDebitReturn);
     technicalPerson.transactions.push(directDebitReturn);
```
Comment (p-janik) [inline src/routes/transactions.ts:107]: oh wow, I assume this account contains a lot of bookings?

**Review by p-janik — APPROVED** [review]

File: src/routes/transactions.ts (hunk 2 above)
Comment (lera) [reply src/routes/transactions.ts:106]: should we update technical account? nobody cares about it.

File: src/routes/transactions.ts (hunk 1 above)
Comment (lera) [reply src/routes/transactions.ts:48]: it gets person by id. It's not using `getAllPersons`

**Pull request merged**